server/src/metakb/services/manage_data.py

```diff
@@ -98,6 +98,24 @@ def is_loadable_statement(statement: Statement) -> bool:
             proposition.subjectVariant,
         )
         success = False
+    else:
+        if len(proposition.subjectVariant.constraints) != 1:
+            _logger.info(
+                "%s could not be loaded because it contains more than 1 constraint: %s",
+                statement.id,
+                proposition.subjectVariant.constraints,
+            )
+            success = False
```

Comment:
will there ever be a case where a subject variant has more than one constraint?

Reply:
Yep! This is a planned future thing but isn't super possible currently. A lot of the parts of this code are basically designed as guards against stuff that isn't supported yet but will be supported eventually